chapters/en/slides/module1/module1_01.Rmd

```diff
@@ -61,242 +64,122 @@ as we will see later.
+    print(
+        df
+        .agg(['mean', 'std']).round(2).reset_index().rename(columns={'index': ''})
+        .to_html(index=False)
+        .replace('class="dataframe"', 'style="width:70px; float:left; margin-right:40px"')
+        .replace('<thead>',  '<thead style="text-align: center">'))
+```
 
 Notes:
-What about if I showed you the commonly used numerical summaries of the data?
-Tough luck,
-all the summary statistics in the table below are **the same** for each of these four set.
+Summaries,
+such as the mean and standard deviation,
+are helpful statistical tools
+that are often useful for detecting difference between datasets.
+However,
+since they collapse the data into just a few numbers,
+statistical summaries can't tell the whole story about the data
+and there can be important differences between datasets
+that summaries fail to reveal.
 
 ---
 
 ## Plotting the data immediately reveals the differences to the human eye
 
-<img src="/module1/anscombes-quartet.png" alt="A caption" width="80%"></img>
+### There are clear patterns in these datasets that we could not detect from looking at the numbers
+
+```{python include=FALSE}
+# Make plot elements easier to see
+# This is a bit ugly but there is no faster why,
+# maybe we should discuss if we want this in all slide decks
+# or just go with the default smaller font.
+def bigger_font():
+    return {
+        'config': {
+            'view': {'continuousWidth': 400, 'continuousHeight': 300},
+            'legend': {'symbolSize': 14, 'titleFontSize': 14, 'labelFontSize': 14},
+            'axis': {'titleFontSize': 14, 'labelFontSize': 12},
+            'header': {'titleFontSize': 16, 'labelFontSize': 14},
+            'encoding': {'x': {'scale': {'zero': False}}}}}
+alt.themes.register('bigger_font', bigger_font)
+alt.themes.enable('bigger_font')
+
+(alt.Chart(anscombe).mark_circle(size=80, opacity=1).encode(
+    x='X',
+    y='Y')
+ .properties(width=300, height=200)
+ .facet('Set', columns=2)
+ .configure_axis(grid=True)
+ .configure_header(title=None)
+ .save(path + 'modified-anscombes.svg'))
+```
+
+<img src="/module1/modified-anscombes.svg" alt="" width="65%"></img>
 
 Notes:
 Humans are not good at detecting patterns in raw numbers,
-and we don't have good intuition about which combination of numbers can contribute to the same statistical summaries.
-But guess what we excel at? Detecting visual patterns!
+and we don't have good intuition about how different distributions of data
+can contribute to identical statistical summaries.
+But guess what we excel at?
+Detecting visual patterns!
 It is immediately clear to us how these sets of numbers differ
-once they are shown as graphical objects
+once they are shown as graphical elements
 instead of textual objects.
-And this is one of the main reasons why data visualization
+This is one of the main reasons why data visualization
 is such a powerful tool for data exploration and communication.
 
```

Comment:
Maybe spend some time talking about how these sample distributions differ, and how that could matter?